fecgov/fec-cms
# [HOTFIX - DO NOT MERGE] set default chart office to S from P

## Summary

- Resolves #4418 

## Impacted areas of the application

Updated the default office for the homepage fundraising chart (top entities) and Raising and Spending by the numbers

## Screenshots

![image](https://user-images.githubusercontent.com/26720877/108904507-93f57200-75ec-11eb-9384-a25fe8df19d8.png)

![image](https://user-images.githubusercontent.com/26720877/108904565-a96a9c00-75ec-11eb-8f5e-fc83a597f372.png)

![image](https://user-images.githubusercontent.com/26720877/108904411-732d1c80-75ec-11eb-879f-1f68377d9f93.png)


## Related PRs

None

## How to test

- pull the branch
- `npm i` (just in case)
- `npm run build` (just in case)
- `./manage.py runserver`
- Check the [homepage Who's raising the most](http://127.0.0.1:8000). Default office should be Senate
- Click 'Browse top raising candidates' 
- [Raising by the numbers](http://127.0.0.1:8000/data/raising-bythenumbers/) Aggregate totals and Who's raising the most should default to Senate races
- Same with [Spending by the numbers](http://127.0.0.1:8000/data/spending-bythenumbers/)

**DO NOT MERGE — HOTFIX**

____
